src/middleware/README.md

```diff
@@ -16,6 +16,6 @@ require('http').createServer(middleware).listen(3000)
 
 ## API
 
-The `middleware` API implements [`.on()`](../#webhookson) and [`.removeListener()`](../#webhooksremovelistener).
+The `middleware` API implements [`.on()`](https://github.com/octokit/webhooks.js/blob/HEAD/README.md#webhookson) and [`.removeListener()`](https://github.com/octokit/webhooks.js/blob/HEAD/README.md#webhooksremovelistener).
```

Comment:
```suggestion
The `middleware` API implements [`.on()`](../README.md#webhookson) and [`.removeListener()`](../README.md#webhooksremovelistener).
```

I think that should fix the issue :)

Reply:
Almost! Your exact suggestion doesn't work for me but I think `../../README.md` does 🤔

Reply:
Yup, that's my math being wrong - we're 2 directories down, not one 😅